lib/nerves_firmware_ssh/fwup.ex

```diff
@@ -94,6 +95,10 @@ defmodule Nerves.Firmware.SSH.Fwup do
     {:noreply, %{state | port: nil}}
   end
 
+  def handle_info({:DOWN, _, :process, cm, reason}, %{cm: cm} = state) do
```

Comment:
What do you think about a `send(port, {self(), :close})` here to make sure `fwup` stops as soon as it can?

Reply:
That is probably a good idea - I have added it.